client-tez/pom.xml

```diff
@@ -1,20 +1,4 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!--
```

Comment:
Why do you remove the license? we must include the Apache License in every file.

Reply:
ok, we added it